openwisp_users/admin.py

```diff
@@ -217,11 +221,14 @@ def save_model(self, request, obj, form, change):
         """
         super(UserAdmin, self).save_model(request, obj, form, change)
         if obj.email:
-            EmailAddress.objects.add_email(request,
-                                           user=obj,
-                                           email=obj.email,
-                                           confirm=True,
-                                           signup=True)
+            try:
+                EmailAddress.objects.add_email(request,
+                                               user=obj,
+                                               email=obj.email,
+                                               confirm=True,
+                                               signup=True)
+            except smtplib.SMTPSenderRefused as e:
+                logger.exception(e)
```

Comment:
`logger.exception()` will automatically log the stack trace of the exception being captured, the argument passed to the method is used to add additional information, so passing it `e` won't be very useful.

Change this line to:

```python 
logger.exception(
    'Got exception {} while sending '
    'verification email to user {}, email {}'.format(
        type(e), obj.username, obj.email
    )
)
```